modules/development/pages/configuration.adoc

```diff
@@ -0,0 +1,37 @@
+= Configuration
+:toc:
+:toclevels: 3
+
+== Software
+
+In your VM or bare metal hypervisor host, you will need to install the following software:
```

Comment:
```suggestion
The hypervisor host requires the following software:
```

Instead of constantly saying "VM or bare metal hypervisor" just refer to it as the hypervisor. Either way, it's going to be the same software and we can use less words.

Reply:
fewer words used.